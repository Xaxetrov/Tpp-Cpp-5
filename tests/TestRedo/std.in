S s1 -5 5 -15 -10
R r1 0 0 10 10
LIST
UNDO
LIST
PC pc1 0 0 0 10 10 10
PC pc2 1 1 1 11 11 11
LIST
UNDO
UNDO
UNDO
LIST
PC pc1 0 0 0 10 10 10 10 0
HIT pc1 5 5
MOVE pc1 20 0
HIT pc1 5 5
UNDO
HIT pc1 5 5
EXIT
