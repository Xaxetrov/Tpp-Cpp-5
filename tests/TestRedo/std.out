C: S s1 -5 5 -15 -10
R: OK
C: R r1 0 0 10 10
R: OK
C: LIST
Rectangle named r1
Segment named s1
R: OK
C: UNDO
R: OK
C: LIST
Segment named s1
R: OK
C: PC pc1 0 0 0 10 10 10
R: OK
C: PC pc2 1 1 1 11 11 11
R: OK
C: LIST
Polygon named pc1
Polygon named pc2
Segment named s1
C: UNDO
R: OK
C: UNDO
R: OK
C: UNDO
R: OK
C: LIST
R: OK
C: PC pc1 0 0 0 10 10 10 10 0
R: OK
C: HIT pc1 5 5
R: YES
C: MOVE pc1 20 0
R: OK
C: HIT pc1 5 5
R: NO
C: UNDO
R: OK
C: HIT pc1 5 5
R: YES
C: EXIT
